README.md

```diff
@@ -58,12 +58,14 @@ file. If you changed the properties of the postgres db, you also have to change
 
 ### Run
 
-#### Docker-compose
+#### Run with Docker-compose
 
 Start all dependencies with our docker-compose files.
-Check
-the [manual for starting the dependencies with docker-compose](https://github.com/Gamify-IT/docs/blob/main/dev-manuals/languages/docker/docker-compose.md)
-.
+Check the [manual for docker-compose](https://github.com/Gamify-IT/docs/blob/main/dev-manuals/languages/docker/docker-compose.md).
+
+To run the main branch with minimal dependencies use the `docker-compose.yaml` file.\
+To run the latest changes on any other branch than `main` use the `docker-compose-dev.yaml` file.
```

Comment:
thats not the intention of these dockerfiles
the `docker-compose.yaml` file is for running the local checkout together with the minimal dependencies for testing.
the `docker-compose-dev.yaml` is for only running the dependencies, where one can run the backend on the host in order to attach a debugger.